### PR TITLE
Make benchmarks great again

### PR DESCRIPTION
**Pull request by roversch**
Benchmarks were a bit broken..

Some fixes and improvements are in this PR

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,19 +5,19 @@ BENCH_EIGEN = build/bin/bench-eigen
...
 # run-benchmark-blaze-dynamic:
-# 	$(BENCH_TMPC) --benchmark_filter="BM_gemm_blaze_dynamic<double>*" $(BENCHMARK_OPTIONS) \
+# 	$(BENCH_BLAST) --benchmark_filter="BM_gemm_blaze_dynamic<double>*" $(BENCHMARK_OPTIONS) \
 # 		--benchmark_out=${BENCH_DATA}/blaze-dynamic.json
 
-${BENCH_DATA}/loop-naive.json:
-	$(BENCH_TMPC) --benchmark_filter="BM_gemm_loop_naive<double>*" $(BENCHMARK_OPTIONS) \
-		--benchmark_out=${BENCH_DATA}/loop-naive.json
+# ${BENCH_DATA}/loop-naive.json:
+# 	$(BENCH_BLAS) --benchmark_filter="BM_gemm_loop_naive<double>*" $(BENCHMARK_OPTIONS) \
```
Comment (roversch) [inline Makefile:19]:
```suggestion
# 	$(BENCH_BLAST) --benchmark_filter="BM_gemm_loop_naive<double>*" $(BENCHMARK_OPTIONS) \
```

File: Makefile [hunk 2]
```diff
@@ -65,18 +65,22 @@ ${BENCH_DATA}/sgemm-blaze-static.json: $(BENCH_BLAZE)
 ${BENCH_DATA}/${BENCH_BLAST_OUTPUT_DIR}/dgemm-blast-static-panel.json: $(BENCH_BLAST)
 	$(BENCH_BLAST) --benchmark_filter="BM_gemm_static_panel<double, .+>" $(BENCHMARK_OPTIONS) \
 		--benchmark_out=${BENCH_DATA}/${BENCH_BLAST_OUTPUT_DIR}/dgemm-blast-static-panel.json
+	cp $@ ${BENCH_DATA}
```
Comment (mkatliar) [inline Makefile:68]: This copies `${BENCH_DATA}/${BENCH_BLAST_OUTPUT_DIR}/dgemm-blast-static-panel.json` to `${BENCH_DATA}/dgemm-blast-static-panel.json`, do I get it right? Why is it needed?

File: Makefile [hunk 3]
```diff
@@ -65,18 +65,22 @@ ${BENCH_DATA}/sgemm-blaze-static.json: $(BENCH_BLAZE)
...
 
 ${BENCH_DATA}/${BENCH_BLAST_OUTPUT_DIR}/dgemm-blast-dynamic-panel.json: $(BENCH_BLAST)
 	$(BENCH_BLAST) --benchmark_filter="BM_gemm_dynamic_panel<double>" $(BENCHMARK_OPTIONS) \
 		--benchmark_out=${BENCH_DATA}/${BENCH_BLAST_OUTPUT_DIR}/dgemm-blast-dynamic-panel.json
+	cp $@ ${BENCH_DATA}
```
Comment (mkatliar) [inline Makefile:73]: Same as above -- why the copy?

File: Makefile (hunk 2 above)
Comment (roversch) [reply Makefile:68]: This is for the Python script. I copy everything into the same 'main' folder.

**Pull request merged**